config/colorterms.py

```diff
@@ -18,6 +18,7 @@
         'HSC-Y': Colorterm(primary="z", secondary="i", c0=0.003386, c1=0.428877, c2=0.076738),
         'IB0945': Colorterm(primary="z", secondary="i", c0=0.008117, c1=0.234991, c2=-0.042255),
         'NB0387': Colorterm(primary="u", secondary="g", c0=-0.709229, c1=0.310719, c2=-0.044107),
+        'NB0395': Colorterm(primary="u", secondary="g", c0=-0.484711, c1=-0.095468, c2=0.002404),
```

Comment:
Were these color terms derived the same way as the others? I have a vague recollection of someone being unsure of the provenance of some of them.

Reply:
These color terms are taken verbatim from the hscPipe 8.x patch script circulated amongst the group in Japan (attached to the Jira ticket). I've spot checked a number of the bands that we've already defined and those that are similarly defined in the patch script, and those numbers are identical, so I'm fairly confident that these numbers should be appropriate here too.